src/Microsoft.Extensions.ServiceDiscovery.Abstractions/Microsoft.Extensions.ServiceDiscovery.Abstractions.csproj

```diff
@@ -9,6 +9,10 @@
     <RootNamespace>Microsoft.Extensions.ServiceDiscovery</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <MinCodeCoverage>82</MinCodeCoverage>
```

Comment:
nit: this can be in the `PropertyGroup` above.

Reply:
I wanted to keep `MinCodeCoverage` in own group for ease of rebase. I already had to perform few of those and deal with few merge conflicts. If I added the new property to the existing groups, the conflicts would be much harded to deal with.